roles/openshift_node/tasks/upgrade.yml

```diff
@@ -7,6 +7,18 @@
 
 # tasks file for openshift_node_upgrade
 
+- name: Delete OpenShift SDN pods prior to upgrade
```

Comment:
A comment referencing the bug and explaining why we do this would be nice.

Reply:
Agreed.  Updating.